src/pkg/clouds/gcp/iam.go

```diff
@@ -23,7 +23,7 @@ import (
 func (gcp Gcp) EnsureRoleExists(ctx context.Context, roleId, title, description string, permissions []string) (string, error) {
 	client, err := iamadm.NewIamClient(ctx)
 	if err != nil {
-		return "", fmt.Errorf("iam.NewIamClient: %w", err)
+		return "", fmt.Errorf("unable to check role exists, failed to create Iam client: %w", err)
```

Comment:
I'm inclined to use the same word as the method name "ensure" instead of "check"
```suggestion
		return "", fmt.Errorf("unable to ensure role exists, failed to create Iam client: %w", err)
```